be/src/runtime/runtime_state.h

```diff
@@ -359,12 +379,16 @@ class RuntimeState {
 
     // put here to collect files??
     std::vector<std::string> _output_files;
-    std::atomic<int64_t> _num_rows_load_total{0};      // total rows read from source
+    std::atomic<int64_t> _num_rows_load_source_total{0};  // total rows read from load source
+    std::atomic<int64_t> _num_bytes_load_source_total{0}; // total bytes read from load source
+
+    std::atomic<int64_t> _num_rows_load_sink_total{0};  // total rows read from load sink
```

Comment:
I think the name `_num_rows_load_sink_total` is little confuse. and can you add some comment explain the difference between `_num_rows_load_source_total` and `_num_rows_load_sink_total`, why need it.